### PR TITLE
New Committer: Senmiao Liu

### PR DESCRIPTION
**Pull request by lsm1**

### CODE REVIEW

**Review by ulysses-you — APPROVED** [review]
Congrats!

**Review by pan3793 — APPROVED** [review]
Congrats

File: src/en/become_committer.md [hunk 1]
```diff
@@ -196,6 +196,12 @@ menu:
       <td><a href="https://github.com/zhouyifan279">zhouyifan279</a></td>
       <td>NetEase</td>
     </tr>
+    <tr>
```
Comment (cxzl25) [inline src/en/become_committer.md:199]: nit alphabetical order

**Review by cxzl25 — APPROVED** [review]
Congratulations!

**Comment by cfmcgrady** [comment]
Congrats!

**Pull request merged**